sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/AppendBlobClient.java

```diff
@@ -588,4 +595,13 @@ public int getMaxAppendBlockBytes() {
             return MAX_APPEND_BLOCK_BYTES_VERSIONS_2022_11_02_AND_ABOVE;
         }
     }
+
+    /**
+     * Get the maximum number of blocks allowed in an append blob.
+     *
+     * @return the max number of blocks that can be uploaded in an append blob.
+     */
+    public int getMaxBlocks() {
+        return MAX_BLOCKS;
```

Comment:
This should return the non-deprecated and non-public constant - `MAX_APPEND_BLOCKS`.